MCprep_addon/mcprep_ui.py

```diff
@@ -1969,6 +1984,25 @@ class McprepProps(bpy.types.PropertyGroup):
 	effects_list_index: bpy.props.IntProperty(default=0)
 
 
+class MCprepWindowManager(spawn_util.VariationProp, bpy.types.PropertyGroup):
+	skin_modes : bpy.props.EnumProperty(
+		name="Modes",
+		description="Skinswap modes",
+		items=(
+			('PLAYER', "Player", ""),
+			('MOB', "Mob/Entity", "")
```

Comment:
These also need to use i18n functions
```suggestion
		name=env._("Modes"),
		description=env._("Skinswap modes"),
		items=(
			('PLAYER', env._("Player"), ""),
			('MOB', env._("Mob/Entity"), "")
```